totalRP3/Modules/Dashboard/Dashboard.lua

```diff
@@ -73,6 +74,24 @@ function TRP3_API.dashboard.getCharacterExchangeData()
 	return get("player/character");
 end
 
+local function onWalkupChange(walkup)
+	local character = get("player/character");
+	local old = character.WU;
+	character.WU = walkup;
+	if old ~= walkup then
+		incrementCharacterVernum();
+	end
+end
+
+local function switchWalkup()
+	if get("player/character/WU") == TRP3_Enums.WALKUP.NO then
+		onWalkupChange(TRP3_Enums.WALKUP.YES);
+	else
+		onWalkupChange(TRP3_Enums.WALKUP.NO);
+	end
+end
+TRP3_API.dashboard.switchWalkup = switchWalkup;
+
```

Comment:
Not much point in adding these when the Player class has an appropriate setter I feel.